torchvision/models/squeezenet.py

```diff
@@ -96,13 +105,13 @@ def __init__(self, version='1_0', num_classes=1000):
                 if m.bias is not None:
                     init.constant_(m.bias, 0)
 
-    def forward(self, x):
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
         x = self.features(x)
         x = self.classifier(x)
         return torch.flatten(x, 1)
 
 
-def _squeezenet(version, pretrained, progress, **kwargs):
+def _squeezenet(version: str, pretrained: bool, progress: bool, **kwargs) -> SqueezeNet:
```

Comment:
nit: you are not typing `kwargs` here.

Also, should we start enabling mypy tests for each one of the files that we enable?

Reply:
Nice catch! Just fixed it, that would be good, my local mypy run didn't catch this.
But I'm unsure about how to proceed with this? Any suggestion about the `mypy.ini`?

Reply:
According to @pmeier, the only tests currently disabled are in https://github.com/pytorch/vision/blob/aa4cf03961f93fd8290b8a57ca1052d01dfc0f5c/mypy.ini#L15-L25 so I'm not sure why it wasn't caught before.

Reply:
This did not show up since we don't enforce annotations yet. If we hit a significant coverage we should enable [`disallow_untyped_defs`](https://mypy.readthedocs.io/en/stable/config_file.html#confval-disallow_untyped_defs). Unfortunately, `mypy` makes no difference between an untyped and partially typed definition and  thus this is disabled for now.